source/styles/base/fonts.styl

```diff
@@ -1,25 +1,25 @@
 @font-face
 	font-family 'Roboto'
 	src local('Roboto'),
-		url('fonts/roboto/regular.woff2') format('woff2'),
-		url('fonts/roboto/regular.woff') format('woff')
+		url('/static/fonts/roboto/regular.woff2') format('woff2'),
+		url('/static/fonts/roboto/regular.woff') format('woff')
 	font-style normal
 	font-weight 400
 
 
 @font-face
 	font-family 'Roboto'
 	src local('Roboto'),
-		url('fonts/roboto/medium.woff2') format('woff2'),
-		url('fonts/roboto/medium.woff') format('woff')
+		url('/static/fonts/roboto/medium.woff2') format('woff2'),
+		url('/static/fonts/roboto/medium.woff') format('woff')
 	font-style normal
 	font-weight 500
 
 
 @font-face
 	font-family 'Roboto'
 	src local('Roboto'),
-		url('fonts/roboto/bold.woff2') format('woff2'),
-		url('fonts/roboto/bold.woff') format('woff')
+		url('/static/fonts/roboto/bold.woff2') format('woff2'),
+		url('/static/fonts/roboto/bold.woff') format('woff')
```

Comment:
I wanted to fix it myself, but you've made PR from your own fork and I can’t commit to it. Here `/static` prefix is unnecessary because after build  these files will be moved to the site’s root and their URL will be like `/fonts/roboto/...`.